README.md

```diff
@@ -1 +1,5 @@
-# self-code-review
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Great job adding the Readme file, but next time kindly try to be more descriptive about your project, like adding a title, description, authors, etc. Please have a look to this [Readme template](https://github.com/microverseinc/readme-template) provided by Microverse and save it for yourself as a reference. 😉

Reply:
**[OPTIONAL]:** Awesome job pointing out the requested change and using respectful yet precise language! :necktie: However, I would highly suggest adding a pro-consequence of implementing this change in the project. In this way, the author of the PR can have a clear idea of the importance of making this change :heavy_check_mark: